### PR TITLE
bitrise-step-upload-assets-to-github-release 1.0.0

### PR DESCRIPTION
**Pull request by jasperroel**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

Note: there is no icon for this plugin.
# Testing
For testing, you need:
-  a valid "Github Personal Action Token" with `repo` permissions
- a valid "README.md" with content in your _tmp folder
- a valid repository/release to attach this file to

### CODE REVIEW

File: steps/bitrise-step-upload-assets-to-github-release/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
```
[the remark this answers is not in the document]
Comment (jasperroel) [reply steps/bitrise-step-upload-assets-to-github-release/1.0.0/step.yml]: Ah, really? I thought it was either apt or brew. I can definitely do both.
For the review process, does that mean pushing a new version completely?

[the remark this answers is not in the document]
Comment (jasperroel) [reply steps/bitrise-step-upload-assets-to-github-release/1.0.0/step.yml]: I've pushed a new version of step.yml with the `apt` dependencies included (matching the `brew` ones).

[the remark this answers is not in the document]
Comment (koral--) [reply steps/bitrise-step-upload-assets-to-github-release/1.0.0/step.yml]: > For the review process, does that mean pushing a new version completely?

Technically not necessarily, but usually it is easier to just add tag and re-share step.

One PR must not contain more than 1 file and 1 commit (except icon).

[the remark this answers is not in the document]
Comment (jasperroel) [reply steps/bitrise-step-upload-assets-to-github-release/1.0.0/step.yml]: > One PR must not contain more than 1 file and 1 commit (except icon).

Understood. I'll close this one as I created a new one with a bumped version here: https://github.com/bitrise-io/bitrise-steplib/pull/2337

Thank you for walking me through this!

**Comment by jasperroel** [comment]
`1.0.1` PR opened here: https://github.com/bitrise-io/bitrise-steplib/pull/2337

**Pull request closed without merging**